packages/components/src/Filter/FilterButton/FilterButton/FilterButton.module.css

```
@@ -0,0 +1,28 @@
+.filterButton {
+  gap: var(--spacing-xs);
```

Since we're updating, might as well use the new variables :)
```suggestion
  gap: var(--spacing-6);
```